web/hooks/github.go

```diff
@@ -47,11 +47,31 @@ func (wh GitHubWebHook) Handle(w http.ResponseWriter, r *http.Request) {
 	case *github.PushEvent:
 		wh.logger.Debug(log.IndentJson(e))
 		wh.handlePush(e)
+	case *github.InstallationEvent:
+		wh.logger.Debug(log.IndentJson(e))
+		wh.handleInstallation(e)
 	default:
 		wh.logger.Debugf("Ignored event type %s", github.WebHookType(r))
 	}
 }
 
+func (wh GitHubWebHook) handleInstallation(payload *github.InstallationEvent) {
+	wh.logger.Debugf("Received app installation event with ID %s for organization %s with ID %d",
+		payload.Installation.ID, *payload.Sender.Login, payload.Installation.Account.ID)
+
+	// TODO: check if course for such organization name exists in the database
+	course, err := wh.db.GetCourse(uint64(*payload.Installation.Account.ID), false)
```

Comment:
We should probably use the payload getter methods (I think they protect better against nil panics), similar to what we do in `handlePush` below.

Reply:
Like these:
```go
	payload.GetInstallation().GetID()
	payload.GetInstallation().GetAccount().GetID()
	payload.GetSender().GetLogin()
```